larq_compute_engine/mlir/tests/optimize_target_arm.mlir

```diff
@@ -1,4 +1,34 @@
-// RUN: lce-tf-opt %s -tfl-optimize-lce -verify-diagnostics | FileCheck %s
+// RUN: lce-tf-opt %s -tfl-optimize-lce-target-arm -verify-diagnostics | FileCheck %s
+
+///////////////////////////
+// Target-specific tests //
+///////////////////////////
+
+// CHECK-LABEL: @reorder_maxpool_2d_quantize
```

Comment:
This is quite ugly. Basically, there are two sets of `optimize` pass tests, one for the Arm pass and one for the xcore pass. Most of the desired behaviour is the same, so most of the test cases are the same. Unfortunately, there isn't a nice way to define the 'common' test cases in one place since there's no 'include' statement in these MLIR filecheck files. Hence, my current solution is to have these two separate files with almost identical content - I have at least highlighted where they differ with the headings "Target-specific tests" and "Common tests".

Any other ideas are very welcome!

Reply:
I think the current way is fine, the comments make it clear which part is duplicate.
Another option would be is to write something like ` // LCE_INCLUDE: filename` and then let the `run_lit.sh` script inject the file at that position. This should be easy to do from bash with `sed` or maybe even just `cat` to concatenate the files. (We could even replace it by a python script if that's easier, although I think it mainly calls subprocesses which arguably is what bash is made for.)

Reply:
Ah yeah, nice, well if @lgeiger thinks that would be nicer we can do that in this PR, otherwise that's an improvement we can do in a later PR.

Reply:
With help from @lgeiger I was able to get this working in a much nicer way, so resolving this now.